gh-actions/go/generate/action.yaml

```diff
@@ -24,14 +24,29 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - name: Set up jq
+      shell: bash
+      run: |
+        set -eu
```

Comment:
Can you put it after the first `echo` (again as in the other steps in this action). That way, we step name will be the echo line and not `set -eu`

Reply:
ok, done I wanted to protect also a potential error on `echo` :joy:

Reply:
If echo starts to return 1, I think the script is the least of the concern on this machine! :p